dockerfiles/web/Dockerfile-prod

```diff
@@ -0,0 +1,43 @@
+FROM python:3.8-slim-bullseye as base
```

Comment:
I think some general comments here would be helpful.

Reply:
Agreed! I mostly copied this, but now looking at it again, I think I should be able to remove some of these packages. So I will do that first and then add comments why we need each of the remaining pieces.

Reply:
Added some comments now and removed the script that installs sqlite which is unused.